tools/ci/src/commands/compile_check_no_std.rs

```diff
@@ -134,6 +134,14 @@ impl Prepare for CompileCheckNoStdCommand {
             "Please fix compiler errors in output above for bevy_state no_std compatibility.",
         ));
 
+        commands.push(PreparedCommand::new::<Self>(
```

Comment:
We should probably start deduplicating this logic soon.

Reply:
Yeah @BD103 and I had a chat about this earlier today. I like the idea of moving this out into a `toml`, `csv` or some other "plain data" file format.

Reply:
I also think we're rapidly reaching a critical mass of functionality where we could replace this CI task with just an example crate instead.

Reply:
It would be nice if this could be rolled up into a loop, but the macro requires a string literal, which means no format string.